maxim/tests/openai/realtime/context_summarization_with_realtime_api_sdk.py

```
@@ -0,0 +1,459 @@
+def print_history(state) -> None:
+    """Pretty-print the running transcript so far."""
+    print("—— Conversation so far ———————————————")
+    for turn in state.history:
+        text_preview = (turn.text or "").strip().replace("\n", " ")
+        print(f"[{turn.role:<9}] {text_preview}  ({turn.item_id})")
+    print("——————————————————————————————————————————")
+
+
+async def mic_to_queue(pcm_queue: asyncio.Queue[bytes]) -> None:
+    """
+    Capture raw PCM‑16 microphone audio and push ~CHUNK_DURATION_MS chunks
```

Ruff (0.14.4)

Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)